### ISSUE TITLE
[Packer Plugins][ZIP]: can't create archives with custom extension when unpacking and replacing files are OK

### ISSUE THREAD

**Issue by PetbkA** [doublecmd/doublecmd#1897]
### Version

Double Commander
Version: 1.1.19 gamma
Revision: 585
Commit: 9dd31c2e9
Build date: 2024/10/06
Lazarus: 3.7.0.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 10 21H2 x86_64

### Description

I've set ZIP plugin to [work with PKG extension](https://github.com/user-attachments/assets/57170c8b-1793-4799-9787-f8c4c1d5ba0c) with setting *Can create new archives* flag.

Opening, unpacking and even replacing files inside PKG works OK, but if I'll try to create _new_ PKG, I'll get
> Error: Pack to file file.pkg : Archive format unknown

error

**Comment by alexx2000** [comment]
Yes, this is not supported due WCX-API limitation. Zip-plugin supports a several formats. No way to determine which format to use when unknown extension.
